document-store/src/main/java/org/hypertrace/core/documentstore/postgres/utils/PostgresUtils.java

```diff
@@ -0,0 +1,205 @@
+package org.hypertrace.core.documentstore.postgres.utils;
```

Comment:
Note: Trying to make all these functions common between two independent Query interfaces ( older and newer versions).
So, we will have refactor PRs after the implementation of the new Query Interface.